reference/6/Microsoft.PowerShell.Management/New-PSDrive.md

```diff
@@ -33,26 +33,22 @@ However, because temporary drives are known only to PowerShell, you cannot acces
 
 The following features are added to `New-PSDrive` in Windows PowerShell 3.0:
 
-* **Mapped network drives**
-
-  You can use the **Persist** parameter of `New-PSDrive` to create Windows mapped network drives.
+- Mapped network drives.
+  You can use the *Persist* parameter of `New-PSDrive` to create Windows mapped network drives.
   Unlike temporary PowerShell drives, Windows mapped network drives are not session-specific.
   They are saved in Windows and they can be managed by using standard Windows tools, such as File Explorer and Net Use.
   Mapped network drives must have a drive-letter name and be connected to a remote file system location.
-  When your command is scoped locally (no dot-sourcing), the **Persist** parameter does not persist the creation of a **PSDrive** beyond the scope in which the command is running.
+  When your command is scoped locally (no dot-sourcing), the *Persist* parameter does not persist the creation of a **PSDrive** beyond the scope in which the command is running.
   If you are running `New-PSDrive` inside a script, and you want the drive to persist indefinitely, you must dot-source the script.
-  For best results, to force a new drive to persist indefinitely, add the **Scope** parameter to your command, and set its value to Global.
-
-* **External drives**
-
+  For best results, to force a new drive to persist indefinitely, add the *Scope* parameter to your command, and set its value to Global.
```

Comment:
- Property, parameter, and class names should be **bold**

Reply:
Fixed